pkg/lease/service.go

```diff
@@ -52,12 +52,14 @@ type Eventer interface {
 
 // Manager manages all the actions against a lease
 type Manager interface {
```

Comment:
Lease isn't going to have a manager service so this could be deleted.

Reply:
fixed